### ISSUE TITLE
SUBFRENCH should be equivalent to VOSTFR

### ISSUE THREAD

**Issue by kephasdev** [Sonarr/Sonarr#6995]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

When SUBFRENCH is used in the release title, it should be considered as VOSTFR.

Greys.Anatomy.S01E01.SUBFRENCH.1080p.WEB.x264-GROUP match "French Audio" and language is detected as "French", which is not correct.
Greys.Anatomy.S01E01.VOSTFR.1080p.WEB.x264-GROUP doesn't match "French Audio" and language is detected as "English", which is correct (because Grey's Anatomy original language is English).

### Expected Behavior

SUBFRENCH should be considered as VOSTFR.

### Environment

```markdown
- OS: Ubuntu 24.04
- Sonarr: 4.0.7.1863
- Docker Install: Yes (but k3s)
- Using Reverse Proxy: No
- Browser: Firefox 128
- Database: Sqlite 3.44.2
```

### What branch are you running?

Main

### Trace Logs?

No sure if needed
